data-raw/process_data_sets.R

```diff
@@ -272,3 +272,11 @@ pennies_resamples <-
   select(replicate, everything()) %>%
   unnest(cols = c(data))
 usethis::use_data(pennies_resamples, overwrite = TRUE)
+
+
+
+## Adding international powerlifting data
+ipf_lifts <- readr::read_csv("https://raw.githubusercontent.com/rfordatascience/tidytuesday/master/data/2019/2019-10-08/ipf_lifts.csv") %>% 
```

Comment:
Also, if you are using the .csv stored on Github in another repository, what is the purpose of adding the .csv to `data-raw`?